pkgs/top-level/python-packages.nix

```diff
@@ -8121,7 +8121,9 @@ in {
 
   spyder-kernels_0_5 = callPackage ../development/python-modules/spyder-kernels/0.x.nix { };
 
-  sqlalchemy = callPackage ../development/python-modules/sqlalchemy { };
+  sqlalchemy_1_3 = callPackage ../development/python-modules/sqlalchemy/1.3.nix { };
+  sqlalchemy_1_4 = callPackage ../development/python-modules/sqlalchemy/1.4.nix { };
+  sqlalchemy = self.sqlalchemy_1_4;
```

Comment:
```suggestion
  sqlalchemy = callPackage ../development/python-modules/sqlalchemy { };
```

Reply:
`sqlalchemy/default.nix` doesn't exist anymore, so this would break evaluation.